eng/pipelines/typespec-apiview.yml

```diff
@@ -11,9 +11,9 @@ parameters:
     # Please use 'https://apiviewstagingtest.com/PullRequest/DetectAPIChanges' for testing purposes
 
 pr:
-   branches:
-     include:
-       - main
+  branches:
```

Comment:
If you want to allow all you can simply remove this entire block as that is the default. See https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#remarks